# This patch builds the plugins externally for use at run-time
--- a/Makefile
+++ b/Makefile
@@ -1,14 +1,14 @@
 #uncomment the lines below to include those plugins within the MG executable.
-MG_BUILT_INS+=wiimote
+#MG_BUILT_INS+=wiimote
 #MG_BUILT_INS+=steamcontroller
 #MG_BUILT_INS+=example
 #MG_BUILT_INS+=joycon
 
 #uncomment the lines below to build external plugins to be loaded at run time.
-#MG_PLUG_INS+=wiimote
-#MG_PLUG_INS+=steamcontroller
-#MG_PLUG_INS+=example
-#MG_PLUG_INS+=joycon
+MG_PLUG_INS+=wiimote
+MG_PLUG_INS+=steamcontroller
+MG_PLUG_INS+=example
+MG_PLUG_INS+=joycon
 
 #If you need to run "make eventlists" and it failed to find your
 #input header where all the key codes are defined, put the
